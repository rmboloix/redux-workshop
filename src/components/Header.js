import React from 'react';
import AtrapaloLogo from './AtrapaloLogo';

export default () => {
  var options = {weekday: 'long', year: 'numeric', month: 'long', day: 'numeric'};

  return (
    <div style={{backgroundColor:'#de0000'}} className="jumbotron">
      <div className="clearfix">
        <AtrapaloLogo />
        <div className="col-md-8" style={{margin:'0 auto'}}>
          <h1 style={{textAlign: 'right', verticalAlign: 'middle', lineHeight:2, color:'white'}}
              className="right">
            DT - Tech Library
          </h1>
          <span style={{float:'right'}}>{new Date().toLocaleDateString('es-ES', options)}</span>
        </div>
      </div>
    </div>
  );
};