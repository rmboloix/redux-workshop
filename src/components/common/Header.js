import React from 'react';
import AtrapaloLogo from './AtrapaloLogo';
import Row from './Row';

const Header = () => {
  let options = {weekday: 'long', year: 'numeric', month: 'long', day: 'numeric'};

  //#de0000
  return (
    <Row className="header">
      <div className="clearfix">
        <AtrapaloLogo />
        <div className="col-md-8" style={{margin:'0 auto'}}>
          <h1 style={{textAlign: 'right', verticalAlign: 'middle', color:'white'}}
              className="right">
            DT - TECH LIBRARY
          </h1>
          <span style={{float:'right', color:'white'}}>{new Date().toLocaleDateString('es-ES', options)}</span>
        </div>
      </div>
    </Row>
  );
};

export default Header;