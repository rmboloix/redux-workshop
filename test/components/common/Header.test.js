import React from 'react';
import Header from '../../../src/components/common/Header';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  Date = jest.fn(() => {
    return {
      toLocaleDateString: function() {
        return '2017 M04 8, Sat';
      }
    }
  });
  const tree = renderer.create(
    <Header />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
