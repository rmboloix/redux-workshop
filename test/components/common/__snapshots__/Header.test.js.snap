// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="col-md-4"
  >
    <img
      alt="Atrapalo Tech library title"
      height={50}
      src="https://www.atrapalo.com/assets/common/logo-acento@2x.png"
    />
    <img
      alt="Atrapalo Tech library logo"
      height={50}
      src="https://www.atrapalo.com/assets/common/a@2x.png"
    />
  </div>
  <div
    className="col-md-8"
    style={
      Object {
        "margin": "0 auto",
      }
    }
  >
    <h1
      className="right"
      style={
        Object {
          "color": "white",
          "textAlign": "right",
          "verticalAlign": "middle",
        }
      }
    >
      DT - TECH LIBRARY
    </h1>
    <span
      style={
        Object {
          "color": "white",
          "float": "right",
        }
      }
    >
      2017 M04 8, Sat
    </span>
  </div>
</div>
`;
